deps-injector/deps-injector.js

```diff
@@ -0,0 +1,62 @@
+
+require.config({
+    paths: {
+        pumascript: '../src/pumascript'
+    }
+});
+
+
+require(['pumascript'], function(puma){
+    
+    function PumaInjector(dependencies){
+        this.version = "1.0"
```

Comment:
We have the simple quotes style. Please check.

Reply:
Thanks!